pages/blog/oss-contributor-workflow.mdx

```diff
@@ -0,0 +1,105 @@
+import withArticle from '../../ui/blog/article';
+
+export const meta = {
+  title:
+    'Making Open Source Easy - Orchestrating the Open Source Contribution Workflow',
+  author: 'tvvignesh',
+  tags: ['open source', 'contributor', 'workflow'],
+  date: '2021-04-21',
+  description:
+    'Sharing our journey of making it easier for people to participate in open source',
+  image: '/blog-assets/oss-contributor-workflow/cover.png',
+  thumbnail: '/blog-assets/oss-contributor-workflow/cover.png',
+};
+
+export default withArticle({ ...meta });
+
+### TLD;R
+
+- We released [The Guild's contributor guide](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md)
+- [Contributor workflow](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md#a-typical-contributor-workflow)
+- [Video tutorials on how to contribute](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md#contributor-videos)
+- [The Guild's tooling](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md#the-tooling) for maintaining open source libraries
+- A [new system](https://github.com/the-guild-org/shared-resources/) for managing our repositories (Process based labeling system, issue templates)
+
+Open source has revolutionized the way we build, collaborate and release software and there are definitely a lot of instances where the community of contributors and maintainers have taken the lead to make the projects thrive in the ecosystem and this is really exciting to see.
+Making open source more approachable has beneficial for the maintainers, users and the contributors themselves - opening new paths for them that seemed harder before.
```

Comment:
is beneficial?  has benefits?
It's worth trying opening new paths?
something like that?